site/app/Resources/views/user/settings_notification.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+Template for the user to edit notification settings
+page path: /user/settings/notification-settings
+#}
+{% extends 'user/settings_base.html.twig' %}
+
+{# Inform settings_base.html.twig about the page currently displayed #}
+{% set settings_subpage = 'notification' %}
+
+{% block settings_content %}
+    <h2>Notification Settings</h2>
+    <p> By default, all users get notifications from the Librecores community in the form of web notifications
+        on the website as well as email notifications to their registered email. The settings for each of these notifications
+        is down to personal preferences and can be edited in the panel down below</p>
+    <h4>Email Notifications</h4>
+    {# Set global status of user subscription to emails beforehand #}
+    {% set enStatus = user.subscribedToEmailNotifs ? "IS_SUBSCRIBED" : "IS_NOT_SUBSCRIBED" %}
+    <ul>
+        <li>Customize if you would like to recieve email notifications from Librecores</li>
+        {% if enStatus == "IS_SUBSCRIBED" %}
+            <form method="post" action="{{ path('librecores.user.settings.notification.email', {status: enStatus}) }}">
+                <button class="btn btn-danger" type="submit">Turn Off</button>
+            </form>
+        {% else %}
+            <form method="post" action="{{ path('librecores.user.settings.notification.email', {status: enStatus}) }}">
+                <button class="btn btn-success" type="submit">Turn On</button>
+            </form>
+        {% endif %}
```

Comment:
Please make this a normal form with a checkbox to enable/disable notifications, and submit this form by clicking on a "submit" button. Use the symfony form component instead of manually writing a HTML form.